data/hoverboard.config.json

```diff
@@ -55,5 +57,45 @@
     "lg": {
       "min": "1024px"
     }
-  }
+  },
+  "socialNetwork": {
+    "follow" : [
+    {
+      "name": "facebook",
+      "url": "https://www.facebook.com/devfest.ukraine/"
+    },
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/devfest_ukraine/"
+    },
+    {
+      "name": "instagram",
+      "url": "https://www.instagram.com/devfestukraine"
+    },
+    {
+      "name": "youtube",
+      "url": "https://www.youtube.com/playlist?list=PLt8lEzcLNl31AouwDkbLbLARlQFwNS2en"
+    }
+  ],
+    "share" : [
```

Comment:
Does it make sense to keep this values here? Most probably users will not change them